P/Python/build_tarballs.jl

```diff
@@ -91,7 +96,7 @@ filter!(!Sys.iswindows, platforms)
 # The products that we will ensure are always built
 products = Product[
     ExecutableProduct(["python", "python3"], :python),
-    LibraryProduct(["libpython3", "libpython3.8"], :libpython),
+    LibraryProduct(["libpython3", "libpython$(version.major).$(version.minor)"], :libpython),
```

Comment:
Is the second option actually necessary since it was wrong?

Reply:
It seems no, `libpython3` matches `libpython3.X`, so I guess we could also just ditch `, "libpython3.8"`.

Reply:
Yes, I'd prefer to simplify to what's actually needed, so
```julia
    ExecutableProduct("python3", :python),
    LibraryProduct("libpython3", :libpython),
```